### PR TITLE
Add OncoKB trademark text & Update dependencies

### PR DESCRIPTION
**Pull request by zhx828**
- This is related to the OncoKB's effort to add trademark to the references(https://github.com/oncokb/oncokb-pipeline/issues/24)
- Updating the dependencies to resolve inaccurate annotation when cancer type is not available

### CODE REVIEW

**Review by leexgh — APPROVED** [review]
LGTM!

File: src/component/variantPage/biologicalFunction/Oncokb.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ interface IOncokbProps {
 export enum ONCOGENICITY {
     ONCOGENIC = 'Oncogenic',
     LIKELY_ONCOGENIC = 'Likely Oncogenic',
-    PREDICTED_ONCOGENIC = 'Predicted Oncogenic',
+    RESISTANCE = 'Resistance',
```
Comment (onursumer) [inline src/component/variantPage/biologicalFunction/Oncokb.tsx:17]: Ideally we should be getting these type of constants from an external OncoKB package or something like that.

**Review by onursumer — APPROVED** [review]
:raised_hands:

File: src/component/variantPage/biologicalFunction/Oncokb.tsx (hunk 1 above)
Comment (zhx828) [reply src/component/variantPage/biologicalFunction/Oncokb.tsx:17]: @onursumer oh absolutely. We are going to move OncoKB Card as separate component and transfer the ownership to OncoKB as well. Then we will set up the constants there. oncokb-commons? :)

**Pull request merged**